ui/components/app/transaction-decoding/transaction-decoding.component.js

```diff
@@ -0,0 +1,224 @@
+import React, { useContext, useEffect, useState } from 'react';
```

Comment:
This component is responsible for decoded transaction data, the primarily highlight of this feature/PR:

![Screenshot from 2021-12-01 09-43-33](https://user-images.githubusercontent.com/7499938/144240775-1f436119-66a6-4715-9954-797da09c0b00.png)